src/app/screens/Library/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { View } from 'react-native';
+
+import Book from '@components/Book';
+import cover from '@assets/img_book1.png';
+
+function Library() {
+  return (
+    <View>
+      <Book cover={cover} title="A little bird Told Me" author="Timothy Cross" />
+    </View>
```

Comment:
when it's only a `<View>` in the container you can just use a Fragment like `<>...</>`

Reply:
Awesome, didn't know you could use `fragment` with react native ?